src/badge/src/components/Badge.js

```diff
@@ -0,0 +1,71 @@
+// @flow
+import React from 'react';
+import styled from 'styled-components';
+
+import * as Colors from '../../../colors';
+import { getStyleVariation } from '../../../utils/StyleUtils';
+
+const {
+  GREENS,
+  NEUTRALS,
+  PURPLES,
+  REDS,
+  WHITE,
+} = Colors;
+
+const getBackgroundColor = getStyleVariation('mode', {
+  added: GREENS[0],
+  primary: PURPLES[2],
+  removed: REDS[0],
+  secondary: PURPLES[5],
+  subtle: WHITE,
+}, NEUTRALS[6]);
+
+const getFontColor = getStyleVariation('mode', {
+  added: GREENS[3],
+  primary: WHITE,
+  removed: REDS[3],
+  secondary: PURPLES[1],
+  subtle: PURPLES[2],
+}, NEUTRALS[0]);
+
+const BadgeWrapper = styled.span`
+  background-color: ${getBackgroundColor};
+  border-radius: 32px;
```

Comment:
Why 32px specifically? I know this is an edge case, but if the height of the badge somehow ends up being taller than 32px, the rounded corners won't meet.

I recommend using `50%` instead of `32px` if the desired behavior is for the sides to effectively be a half-circle.

Reply:
I was using `32px` because that is what is used in the Atlaskit badge. This is what `50%` looks like:
<img width="98" alt="Screen Shot 2019-11-20 at 1 44 20 PM" src="https://user-images.githubusercontent.com/32921059/69280770-ee0eb880-0b9b-11ea-96c2-26d603f1bb3e.png">

Reply:
Giulia and I had a discussion and played around with percentages. Turns out they don't work as initially expected since it depends on the length of the contents inside. Short badges would look like circles and really long badges would look like ovals.

We settled on using `em` units so that the badge would still look correct regardless of font-sizes and content length.